src/library/export/trackexportworker.h

```diff
@@ -49,6 +49,11 @@ class TrackExportWorker : public QThread {
         return m_errorMessage;
     }
 
+    //Get the list
+    QList<QString> getMissingTracks() const;
+
+    bool m_canceledByUser = false;
```

Comment:
Can we use stop() instead? Normally we avoid public members. This also needs to become an Atomic, to not being on a risk to be optimized away.